### PR TITLE
Update existing tutorials

### PR DESCRIPTION
**Pull request by ffuuugor**
Updated and rerun existing tutorials + some code changes required to make them work.

See inline comments for more details

### CODE REVIEW

File: opacus/accountants/rdp.py [hunk 1]
```diff
@@ -30,6 +30,9 @@ def step(self, noise_multiplier: float, sample_rate: float):
     def get_privacy_spent(
         self, delta: float, alphas: Optional[List[Union[float, int]]] = None
     ) -> Tuple[float, float]:
+        if not self.steps:
+            return 0, 0
+
```
Comment (ffuuugor) [inline opacus/accountants/rdp.py:35]: If you call `get_privacy_spent` on an empty accountant, it fails with a cryptic exception. 
better to return 0

File: opacus/grad_sample/grad_sample_module.py [hunk 2]
```diff
@@ -98,8 +98,8 @@ def __getattr__(self, item):
                 return submodules[item]
             raise e
 
-    def forward(self, x, *args, **kwargs):
-        return self._module(x, *args, **kwargs)
+    def forward(self, *args, **kwargs):
+        return self._module(*args, **kwargs)
```
Comment (ffuuugor) [inline opacus/grad_sample/grad_sample_module.py:102]: `x` in completely unnecessary, and fails for BERT models from huggingface, which only accept named parameters.

`nn.Module` signature doesn't have `x`

File: opacus/validators/batch_norm.py [hunk 3]
```diff
@@ -11,14 +10,7 @@
...
 
 
-logging.basicConfig(
-    format="%(asctime)s:%(levelname)s:%(message)s",
-    datefmt="%m/%d/%Y %H:%M:%S",
-    stream=sys.stderr,
-)
 logger = logging.getLogger(__name__)
-logger.setLevel(level=logging.INFO)
-
```
Comment (ffuuugor) [inline opacus/validators/batch_norm.py:21]: This messes up with global logging config - I don't think it's a right strategy to configure logger while initializing a module (cc @karthikprasad )

**Comment by karthikprasad** [comment]
LGTM If you haven't already, can you rebase it on top of #284? The tutorial might end up needing some changes.

**Comment by ffuuugor** [comment]
will do

**Comment by ffuuugor** [comment]
@karthikprasad merged latest experimental, everything seem to be working fine

**Review by karthikprasad — APPROVED** [review]

**Pull request merged**